app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,113 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ // From GDS's alphagov/govuk_frontend_toolkit
+ @import 'colours';
+ @import 'font_stack';
+ @import 'measurements';
+ @import 'conditionals';
+ @import 'device-pixels';
+ @import 'grid_layout';
+ @import 'typography';
+ @import 'shims';
+
+ @import 'design-patterns/alpha-beta';
+ @import 'design-patterns/buttons';
+ @import 'design-patterns/breadcrumbs';
+
+ // From GDS's alphagov/govuk_elements
+ @import 'elements/helpers';
+ @import 'elements/reset';
+
+ // Base (unclassed HTML elements)
+ // These are predefined by govuk_template
+ // If you're not using govuk_template, uncomment the line below.
+ // HTML elements, set by the GOV.UK template
+ // @import 'elements/base'
+
+ @import 'elements/layout';
+ @import 'elements/elements-typography';
+ @import 'elements/buttons';
+ @import 'elements/icons';
+ @import 'elements/lists';
+ @import 'elements/tables';
+ @import 'elements/details';
+ @import 'elements/panels';
+ @import 'elements/forms';
+ @import 'elements/forms/form-block-labels';
+ @import 'elements/forms/form-date';
+ @import 'elements/forms/form-validation';
+ @import 'elements/breadcrumbs';
+ @import 'elements/phase-banner';
+ @import 'elements/components';
+
+/////////////////////////////
```

Comment:
Might be worth taking everything below out until we need to put it back in? We might not need these styles anymore, plus they're a bit hacky.